tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -267,7 +321,7 @@ static uint8_t supported_commands(const void *cmd, uint16_t cmd_len,
 }
 
 static uint8_t controller_index_list(const void *cmd, uint16_t cmd_len,
-				     void *rsp, uint16_t *rsp_len)
+					 void *rsp, uint16_t *rsp_len)
```

Comment:
Same thing here. Just remove all these indentation changes, since they're unrelated and wrong.